config/routes.rb

```diff
@@ -2,12 +2,13 @@
   resources :posts
 
   devise_for :users,
-            controllers: {
-                sessions: 'users/sessions',
-                registrations: 'users/registrations'
-            }
+          controllers: {
+            sessions: 'users/sessions',
+            registrations: 'users/registrations'
+          }
   get '/member-data', to: 'members#show'
   get '/users/:id', to: 'users#show'
```

Comment:
better to use resources instead of writint every route

Reply:
there you can set actions that would be used in app http://rusrails.ru/routing#nastroyka-marshrutizatora-rails